src/material-experimental/mdc-chips/chips.scss

```diff
@@ -16,6 +14,14 @@
   // Required for the ripple to clip properly in Safari.
   transform: translateZ(0);
 
+  &._mat-animation-noopable {
+    // MDC's chip removal works by toggling a class on the chip, waiting for its transitions
+    // to finish and emitting the remove event at the end. The problem is that if our animations
+    // were disabled via the `NoopAnimationsModule`, the element won't have a transition and
+    // `transitionend` won't fire. We work around the issue by assigning a very short transition.
+    transition-duration: 1ms;
```

Comment:
@crisbeto in Chrome at least, you could make this even smaller, e.g.  `0.0001ms`

Reply:
Anything less than 16ms is less than a frame at 60fps so people problably wouldn't notice it anyway.

Reply:
Yeah I'm not sure if it will make any actual difference, I just want to make sure the delay is as small as possible since animations are _supposed_ to be disabled. It's very possible though that the browser treats `0.0001ms` and `1ms` as the exact same thing